### ISSUE TITLE
Current README in master does not match current release

### ISSUE THREAD

**Issue by WillGibson** [Marak/faker.js#803]
I have got myself confused several times recently when I went to the GitHub page to look up the API methods because it doesn't match the 4.1.0 release any more.

I've made a PR to add a temporary link to the older README to help people like me until 5.0.0 is released.

https://github.com/Marak/faker.js/pull/802

**Comment by jougene** [comment]
Yes - I need some functionality mentioned in README but can get it in 4.1.0

**Comment by aegixx** [comment]
Agreed - this is confusing, given the last official release was almost a year ago.  @Marak anything we can do to help with 5.0.0 release?  (or an interim 4.X release)

**Comment by ardeois** [comment]
I agree, or maybe a `next` release if master is still unstable

**Comment by john-osullivan** [comment]
I'm looking to plug in a mock data generator right now, I'm between `faker.js` and `casual`.  Faker has far, far more users, but its `npm` package hasn't been updated in 3 years, whereas `casual` was published just 5 months ago.  Not sure where the release from a year ago is, there isn't a tag on the repo either.

Now that I know the docs are out of sync, I'm inclined to go with `casual` 🤷‍♂

**Comment by ohcibi** [comment]
A readme evolves along with its repository. This is normal "behavior" for version controlled code. If one wants to access the 4.1.0 (or any other version) readme its as easy as selecting that tag from the branch/tag selector in the repository view. This works for literally any other repository as well, as long as the versions are git tagged.

<img width="427" alt="Screenshot 2019-11-20 at 19 04 23" src="https://user-images.githubusercontent.com/61993/69264998-b407db80-0bc8-11ea-87d9-3592a9f814e5.png">

**Comment by ardeois** [comment]
@ohcibi Yes you are right. However the last tag dates from Feb 21, 2017, and there was around 50 pull requests merged since then.

It would be nice to have a new npm package published, like v5.0.0 (or `next` if master is not stable). This way the README will actually be more accurate and up to date with the latest npm package.
At the end, this could be fixed once this issue is fixed, but it's locked https://github.com/Marak/faker.js/issues/692

Note that I understand this is an open source project and people work on this repository on their own time, I'm thankful for that! 

@Marak If you need any help for making the v5 a reality let me know maybe I can help?

**Comment by danielsnider** [comment]
+1 for a new release, it's been a while
